src/server/twemcache-rs/src/protocol/admin.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use crate::protocol::{CRLF, CRLF_LEN};
+use bytes::BytesMut;
+use std::borrow::Borrow;
+
+// TODO(bmartin): see TODO for protocol::data::Request, this is cleaner here
```

Comment:
Let's go to HTTP sooner rather than later. I'm getting more excited by the day 😄

Reply:
Happy to prioritize this as a follow-up PR - I'm rethinking the exact deps I took on in rpc-perf/rezolus for this, have some work started on that which can be carried over to Pelikan.